openslides_backend/models/motion_change_recommendation.py

```diff
@@ -0,0 +1,45 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionChangeRecommendation(Model):
+    """
+    Model for motion change_recommendation.
+
+    Reverse fields:
+    - motion_ids
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    # Replacement (0), Insertion (1), Deletion (2), Other (3)
+    type = fields.IntegerField(
+        "The type of this motion change recommendation.", max=3, min=0
+    )
+    other_description = fields.CharField(
+        "The other description of this motion change recommendation."
+    )
+    line_from = fields.PositiveIntegerField(
+        "The line from of this motion change recommendation."
+    )
+    line_to = fields.PositiveIntegerField(
+        "The line to of this motion change recommendation."
+    )
+    rejected = fields.BooleanField(
+        description="If this motion change recommendation is rejected"
+    )
+    internal = fields.BooleanField(
+        description="If this motion change recommendation is internal"
+    )
+    text = fields.TextField(description="The text of the motion change recommendation.")
+    creation_time = fields.TimestampField(
```

Comment:
The creation time should be set to the current timestamp on create.

Reply:
So this has to be set via the update_instance hook and should be removed here.

Reply:
Done, set the current timestamp in update_instance and add tests for it.